public-sector-websites.csv

```diff
@@ -365,15 +365,15 @@ Ministry for Primary Industries,gia.org.nz,No,Public Service department
 Ministry for Primary Industries,maf.govt.nz,Yes,Public Service department
 Ministry for Primary Industries,mpi.govt.nz,No,Public Service department
 Ministry for Primary Industries,nzfsa.govt.nz,Yes,Public Service department
-Ministry for the Environment,climatechange.govt.nz,No,Public Service department
```

Comment:
This needs to stay as the domain still exists and redirects to another location.